components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -390,7 +393,7 @@ export class ReportingComponent implements OnInit, OnDestroy {
   onFiltersClear(_event) {
     const queryParams = {...this.route.snapshot.queryParams};
 
-    const filteredParams = pickBy((_value, key) => {
+    const filteredParams = pickBy((_value, key: ReportingFilterTypes) => {
 
       // Only handles control_tag filters
       if (key.includes('control_tag')) {
```

Comment:
Solid suggestion, I've cleaned this function up - but it needed to be done a little bit differently.  I've made a mesh of your suggestion and the previous code, because there are actually two `control_tag` type values, and the clearAll function won't work properly if it is included because control tags rely on second level filtering.  I've updated the note a bit as well.